### PR TITLE
Add cloudbeaver dockerfile.

### PR DESCRIPTION
**Pull request by JDarDagran**
Signed-off-by: Jakub Dardzinski <kuba0221@gmail.com>

Build CloudBeaver with BQ driver in separate workflow.

Add setting metadata in GCP VM to adjust to custom proxy agent.

Signed-off-by: Jakub Dardzinski <kuba0221@gmail.com>

Update inverting proxy image.

Signed-off-by: Jakub Dardzinski <kuba0221@gmail.com>

Reorganize files. Remove Cloudbeaver build.

Signed-off-by: Jakub Dardzinski <kuba0221@gmail.com>

### CODE REVIEW

File: jupyterlab-dataops/Dockerfile [hunk 1]
```diff
@@ -23,9 +22,8 @@ RUN code-server --install-extension ms-python.python@${VSCODE_PYTHON} \
 COPY resources/vsc/settings.json /root/.local/share/code-server/Machine/
 COPY resources/vsc/.bashrc root/
 
-# dbeaver
-COPY --from=builder-dbeaver /opt/cloudbeaver/ /opt/cloudbeaver/
-COPY resources/cloudbeaver/conf /opt/cloudbeaver/conf
+# cloudbeaver
+COPY --from=gcr.io/getindata-images-public/cloudbeaver-dataops /opt/cloudbeaver/ /opt/cloudbeaver/
```
Comment (Santhin) [inline jupyterlab-dataops/Dockerfile:26]: There is no tag for this image and in line 41 comment suggest that we are using default cloudbeaver and here we are pointing for our custom image with bigquery driver

File: jupyterlab-dataops/Dockerfile [hunk 2]
```diff
@@ -40,6 +38,17 @@ COPY resources/jupyter/ /etc/jupyter/
 # Expose port for Vertex AI compatibility
 EXPOSE 8080
 
+# use default cloudbeaver in non-bigquery target
+FROM base AS default
```
Comment (Santhin) [inline jupyterlab-dataops/Dockerfile:42]: In workflows I cannot see any additional target provider / matrix extended with this name, I mean right now these lines of code are unused.

Comment (Santhin) [reply jupyterlab-dataops/Dockerfile:42]: What is the purpose of the default target?

Does it mean that we want to use workbench in the gcp scenario only for communicating with 3 party databases, etc., and not bigquery?

File: jupyterlab-dataops/Dockerfile (hunk 1 above)
Comment (JDarDagran) [reply jupyterlab-dataops/Dockerfile:26]: I forgot to pin 0.1.0 version tag here.

Comment (JDarDagran) [reply jupyterlab-dataops/Dockerfile:26]: 41 should be in 26 actually, that's missed doubled line

File: jupyterlab-dataops/Dockerfile (hunk 2 above)
Comment (JDarDagran) [reply jupyterlab-dataops/Dockerfile]: I've deleted default target then to leave no doubt how this should work eventually.

**Review by Santhin — APPROVED** [review]
LGTM

**Pull request merged**